initrd-dropbear.service

```diff
@@ -2,14 +2,24 @@
 
 # Provide ssh server in initramfs
 
-# TODO
-# see #21
-# see #22
+# note:
+# mutually exclusive with tinysshd
```

Comment:
Maybe add 

```ini
Before=initrd-tinysshd.service
Conflicts=initrd-tinysshd.service
```
So that even when both are activated (by accident or lack of knowledge) tinysshd always runs in the end.

I hope I correctly understood the man page.

Reply:
no: "protecting users from advanced cruft"